.github/workflows/cherryPick.yml

```diff
@@ -0,0 +1,104 @@
+name: Cherry-pick a pull request
+
+on:
+  workflow_dispatch:
+    inputs:
+      PULL_REQUEST_NUMBER:
+        description: The number of a pull request to CP
+        required: true
+
+jobs:
+  validateActor:
+    runs-on: ubuntu-latest
+    outputs:
+      IS_DEPLOYER: ${{ steps.isUserDeployer.outputs.isTeamMember }}
+    steps:
+      - id: isUserDeployer
+        uses: tspascoal/get-user-teams-membership@baf2e6adf4c3b897bd65a7e3184305c165aec872
```

Comment:
Can we add a test that verifies this works? Maybe @Jag96 can try to cherry pick?

Reply:
Good idea!

Reply:
Testing steps updated 👍